src/arithmetic/int_set.cc

```diff
@@ -252,6 +252,68 @@ inline IntervalSet Combine<ir::Mod>(Analyzer* analyzer,
   return IntervalSet::Everything();
 }
 
+
+template<>
+inline IntervalSet Combine<ir::FloorDiv>(Analyzer* analyzer,
+                                         IntervalSet a,
+                                         IntervalSet b) {
+  if (a->IsSinglePoint() && b->IsSinglePoint()) {
+    return IntervalSet::SinglePoint(floordiv(a->min_value, b->min_value));
+  }
+  if (a->IsEmpty()) return a;
+  if (b->IsEmpty()) return b;
+  if (b->IsSinglePoint()) {
+    if (is_zero(b->min_value)) {
+      LOG(FATAL) << "Divide by zero in CombineInterval Div";
+    }
+    if (is_one(b->min_value)) return a;
+    // no relaxation is needed in here due to set is inclusive
+    if (analyzer->CanProveGreaterEqual(b->min_value, 0)) {
```

Comment:
I guess we have to compare with 1 here to avoid division by zero, like in the next if-branch. (Or compare with 0 and remove the check from the TryConstFold function since `b->min_value == 0` may result from relaxation and shouldn't lead to failure).

Reply:
good point, I added a condition to catch this problem

Reply:
Oh, sorry, I forgot that we are in the `b->IsSinglePoint()` branch, so I guess my comment is meaningless since `b->min_value == 0` definitely leads to division by zero in this case, and failing in TryConstFold shouldn't be a problem.
However, I think it is possible to replace `analyzer->CanProveGreaterEqual(-b->min_value, 1)` with `analyzer->CanProveGreaterEqual(-b->min_value, 0)` in the next branch, for symmetry.

Reply:
OK, I will make the change